billy-portugal/src/main/java/com/premiumminds/billy/portugal/util/SAFTs.java

```diff
@@ -49,24 +48,25 @@ public SAFTs(Injector injector) {
 
 	public InputStream export(UID uidApplication, UID uidBusiness, Date from, Date to,
 							  SAFTVersion version) throws SAFTPTExportException, IOException {
-		return this.export(uidApplication, uidBusiness, from, to, SAFTs.TMP_SAFT, version);
+		File tempFile = File.createTempFile("SAFT", ".xml");
```

Comment:
I'm a bit confused. The file is created to pass the path, then it's created again? Why can't the file itself be passed as an argument?

Reply:
yeah, you are right. This was not the right aproach. 

Changed to use a ByteArrayOutputStream and rebased.